src/debug.ts

```diff
@@ -110,26 +110,33 @@ export class DashmipsDebugSession extends LoggingDebugSession {
     protected async launchRequest(response: DebugProtocol.LaunchResponse, args: LaunchRequestArguments) {
         this.config = args
         this.runInTerminalRequest(...buildTerminalLaunchRequestParams(args))
-        await this.configurationDone.wait(1500)
+        await this.configurationDone.wait(1000)
+        await this.configurationDone.wait(900)
```

Comment:
After testing discovered we need to wait a bit longer to get the program to run without breakpoints set.